Content.Shared/GameObjects/Components/Damage/DamageableComponent.cs

```diff
@@ -305,6 +305,14 @@ public void Heal()
                 return false;
             }
 
+
+                    // While we have healing left...
+                    while (healingLeft < 0 && healThisCycle != 0)
```

Comment:
Don't do this. Smug already made a pr to fix it

Reply:
alright, I'lll remove this change.